docker/docker-compose-server.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.3'
+
+services:
+  postgres:
+    image: postgres:latest
+    volumes:
+    - /mnt/vol/postgresql:/var/lib/postgresql
+    environment:
+      POSTGRES_DB: sourcesdb
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    ports:
+    - 5432:5432
+  server:
+    container_name: codesearch_web_server
+    image: quay.io/aelve/codesearch-web-server
+    volumes:
+    - /mnt/vol/data:/data
+    - /mnt/vol/index:/index
+    depends_on:
+    - postgres
+    command:
+      make serve
+    environment:
+      LOG_LEVEL: DEBUG
+      JAVA_OPTS: "-Dplay.filters.hosts.allowed.1=codesearch.aelve.com -Dplay.filters.hosts.allowed.2=cloud.digitalocean.com"
+      APPLICATION_SECRET: u?:IPZT:Cun;SP>edigHDs:cpUf2cY`<7D>b9lHg98^NsVGn]:eLCIjz1Tt_VyIv
+    ports:
+    - 80:80
```

Comment:
Maybe the forwarding port needs to be changed to a more unusual one? For example `9000`?

Reply:
:80 definitely won't work on a standard machine, not because it's busy but because ports lower than 1024 require admin rights (on Linux at least, probably macOS too).

Reply:
Okay, so what's the status of this?

Ideally we want to run it with something like 9000:80 locally and override it to 80:80 for production. Is it possible?

Reply:
@neongreen Yes it is possible. Do you mean depending on the environment variable?

Reply:
I don't have an opinion about how it should be done as long as I don't have to edit the .yaml file manually on the server.

Reply:
Ok, I understood.